engine/src/cmd/unit_generic.h

```diff
@@ -517,6 +504,7 @@ public: Unit();
 //Uses GFX so only in Unit class
     virtual void Draw( const Transformation &quat = identity_transformation, const Matrix &m = identity_matrix ) {}
     virtual void DrawNow( const Matrix &m = identity_matrix, float lod = 1000000000 ) {}
+    virtual std::string drawableGetName() { return name; };
```

Comment:
All 3 of these should be declared `override`, based on what clang is saying

Reply:
Done